app/controllers/users_controller.rb

```diff
@@ -0,0 +1,52 @@
+class UsersController < ApplicationController
+before_action :navbar_categories, only: [:index]
+before_action :find_user, only: [:show, :dash, :product_dash, :order_dash]
+before_action :only_current_user, only: [:dash, :order_dash]
+
+  def index
+    @users = User.all
+    @cat = Category.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(strong_params)
+    if @user.save
+      session[:user_id] = @user.id
+      flash[:notice] = "Welcome to Quartzy! You are now logged in."
+      redirect_to root_path
+    else
+      render :new
+    end
+  end
+
+  def show
+  end
+
+  def dash
+  end
+
+  def product_dash
+  end
+
+  def order_dash
+    @user = @current_user
+    session[:rev_status] = params[:rev_status] if !params[:rev_status].nil?
+    session[:order_status] = params[:order_status] if !params[:order_status].nil?
```

Comment:
As @acmei said, this is a double negative. It could read if params[:order_status]. It's easy to miss the bang, when reading code.